src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -478,6 +498,16 @@ def load_arguments(self, _):
 
         c.argument('max_size_bytes', help='The new maximum size of the database expressed in bytes.')
 
+        c.argument('compute_model',
+                    arg_type=compute_model_param_type,
+                    help='The compute model of this database. Currently we support Provisioned and Serverless.')
+    
+        c.argument('auto_pause_delay', 
+                    help='Time in minutes after which database is automatically paused. A value of -1 means that automatic pause is disabled.')
+
+        c.argument('min_capacity', 
+                    help='Minimal capacity that database will always have allocated, if not paused')
+
```

Comment:
What arg group are these all in? Should auto_pause_delay & min_capacity be in a new arg group in order to demonstrate that they are both related to serverless?

Reply:
great idea, I grouped them, and added help text in _help.py as well